xml/decoders/NCE_D13SR_TC.xml

```diff
@@ -34,7 +35,8 @@
   <!-- Version 8 - added D408SR V3.6 which has different function mapping then V3.5 -->
   <!-- Version 9 - added Factory Direct Trains Trackmobile -->
   <!-- Version 10 - added BLI Hi-Rail pickup and track inspection sedan -->
-  <!-- Version 10 - added N12K0a and N12K0b plus a few comments/tooltips -->  
+  <!-- Version 10 - added N12K0a and N12K0b plus a few comments/tooltips -->
+  <!-- Version 12 - high Ver ID 38, BLI Plymouth switcher -->  
```

Comment:
@marcfour: I see the comment that you're adding "high Ver ID 38" in line 38, but I don't see the actual change to do that later on in the file.  Did it perhaps not get saved before being committed?